src/renderer/cluster-frame.tsx

```diff
@@ -211,13 +210,11 @@ export class ClusterFrame extends React.Component {
               {this.renderExtensionTabLayoutRoutes()}
               {this.renderExtensionRoutes()}
               <Redirect exact from="/" to={this.startUrl}/>
-              {
-                ClusterPageRegistry
-                  .getInstance()
-                  .redirectEntries
-                  .map(url => <Redirect key={url} from={url} to={this.startUrl} />)
-              }
-              <Route component={NotFound}/>
+              <Route render={({ location }) => {
+                Notifications.error(`Unknown location ${location.pathname}, redirecting to main page.`);
```

Comment:
nit: what is "main page"? What about "overview page"? Covers both cluster overview and workloads overview

Reply:
Sure

Reply:
I'll do that in a follow up